### DISCUSSION TITLE
Tried publishing, website now says 'error 404'

### DISCUSSION THREAD

**Discussion by laurafritch** [flowershow/flowershow#631] [General and Help]
I followed the steps to publish a site. For a while it seemed like it just wasn't updating, since the site just said 'welcome to my flowershow'. Now it has an error 404? What can I do?

**Comment by nightlightouo** [comment]
You may need to update the index.md file located in the content folder.

**Comment by rufuspollock** [comment]
@laurafritch did you move / rename the `index.md` file?
